packages/fx-core/resource/package.nls.json

```diff
@@ -507,6 +512,7 @@
   "core.common.ReceiveApiResponse": "Received API response: %s.",
   "core.importAddin.label": "Import an Existing Outlook Add-ins",
   "core.importAddin.detail": "Upgrade an Add-ins project to the latest app manifest and project structure",
+  "core.importOfficeAddin.label": "Import an Existing Office Add-ins",
```

Comment:
Article not required in label.
Proposed: Import Existing Office Add-ins

Reply:
Same as above. Thanks Supkasar!